src/test/java/uk/gov/hmcts/reform/managecase/service/noc/NoticeOfChangeQuestionsTest.java

```diff
@@ -432,6 +433,22 @@ void shouldThrowErrorNoEventAvailable() {
                 .hasMessageContaining("No NoC events available for this case type");
         }
 
+        @Test
+        @DisplayName("Must return an error when multiple NOC Request events are available in the case for the user")
```

Comment:
Although its present in the integration test, should we test here for when there are no events available ?

Reply:
Oops- just seen the test above, please ignore

Reply:
It's already there :) line #425